include/initStress.h

```diff
@@ -0,0 +1,36 @@
+#ifndef INITSTRESS_H
+#define INITSTRESS_H
+
+#include <array>
+#include <vector>
+
+//! \brief    Class to store initial stress
+//! \details  Includes constructor to make the MaterialPoint
+//! \tparam   Tdim (1D, 2D or 3D)
+
+class InitStress {
+ public:
+  //! \brief  Make constructor for initial stress
+  InitStress(unsigned id, const std::array<double, 6> stress) {
+    id_ = id;
+    stress_ = stress;
+  }
+
+  //! Get the private properties using two functions
+  //! To return id_
+  unsigned id() const { return id_; }
+
+  //! To return stress_
+  std::array<double, 6> stress() const { return stress_; }
```

Comment:
I thought there are 6 Voigt stresses --> sigxx, sigyy, sigzz, tauyz, tauzx, tauxy? No? It is always 6 right?